### PR TITLE
Feat: footer-V3 design

### PR DESCRIPTION
**Pull request by cp-sumi-k**
- Fix hydration mismatch issue
- Add theme-color meta
- New footer design

**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [ ] iPad(safari)
- [x] Tablet (chrome, firefox)

Lighthouse score

Screenshot from 2022-12-29 17-13-21

### CODE REVIEW

File: vue-frontend/src/components/contact/ContactFormNew.vue [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    setOption(option) {
+      this.invest = option;
+      this.showList = false;
+    },
+  },
+};
+</script>
+
+<style lang="postcss" scoped>
+.floating-input {
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/contact/ContactFormNew.vue:356]: Can we not apply this css as inline ??

Comment (cp-sumi-k) [reply vue-frontend/src/components/contact/ContactFormNew.vue:356]: I think its used on may places that's why added it here

Comment (cp-shruti-s) [reply vue-frontend/src/components/contact/ContactFormNew.vue:356]: But i think we display repeated css inline only
According to Jimmy say !!

**Review by cp-priyanka-g — APPROVED** [review]

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 2]
```diff
@@ -3,11 +3,11 @@
     <img
       :src="bg2400"
       :srcset="`${bg400} 400w, ${bg800} 800w, ${bg1200} 1200w, ${bg2400} 2400w`"
-      class="tw-w-full tw-absolute tw-top-0 tw-left-0 tw--z-[1]"
+      class="tw-absolute tw-top-0 tw-left-0 tw-w-full tw--z-[1]"
       alt="canopas-cta"
     />
     <div
-      class="tw-container sm:tw-w-4/5 lg:tw-w-[70%] tw-text-center tw-px-[25px] tw-pt-[20%] md:tw-pt-[10%] tw-pb-[5%] tw-m-auto tw-opacity-1"
+      class="tw-container sm:tw-w-4/5 lg:tw-w-[70%] tw-m-auto tw-px-[25px] tw-pt-[20%] md:tw-pt-[10%] tw-pb-[5%] tw-text-center tw-opacity-1"
```
Comment (cp-shruti-s) [inline vue-frontend/src/components/home-new/CTASection.vue:10]: Replace top-bottom padding with percentage by px

**Pull request merged**